doc/source/new-project-guide/level-5-tutorial.rst

```diff
@@ -220,15 +220,28 @@ We require that each PR includes a news item as a ``<branch-name>.rst`` file und
 
 #. Make a copy of ``news/TEMPLATE.rst`` and rename to ``news/<branch-name>.rst``.
 
+#. (optional) If you are using a Linux shell, you can setup an ``alias`` to make the creation of the news file ready for editing much quicker and easier 
+
+    Add the following line to ``~/.bashrc`` or ``~/.zshrc`` file:
+
     .. code-block:: bash
 
-        cp news/TEMPLATE.rst news/$(git rev-parse --abbrev-ref HEAD).rst
+        alias cpnews="cp news/TEMPLATE.rst news/$(git rev-parse --abbrev-ref HEAD).rst"
+
+    Run the following command to apply the shell configuration.
+    
+    .. code-block:: bash
+        
+        source ~/.bashrc  # if you are using bash
+        source ~/.zshrc  # if you are using zsh
 
-#. (optional) If you are using a Linux shell, you can setup an ``alias`` to make the creation of the news file ready for editing much quicker and easier. Add the following line to ``~/.bashrc`` or ``~/.zshrc`` file and run ``source ~/.bashrc`` or ``source ~/.zshrc`` to apply the changes.
+    Now, you can create a news file by running
```

Comment:
"Now, whenever you want to create a news flie, simply navigate to the top level directory in the project needing news and typing `cpnews` on the command line.  You can then open in an editor the file, which will have the name ``<branch-name>.rst`` (where ``news/<branch-name>`` is replaced by the actual branch name.) and be in the ``news`` directory, and add a description of the edits that were done in this PR.  This should be a user-facing high level summary of the edits done in this PR and will be automatically converted into a the CHANGELOG when the code is released."

Reply:
done - incorporated.